ninja-react/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
Needed to run `npm install` beforehand to install all of the required dependencies

Reply:
Added to README